### PR TITLE
support embedded jubatus

### PR DESCRIPTION
**Pull request by kmaehashi**
This PR adds support for "embedded" jubatus.

The new option ``embedded=True`` automatically creates new embedded instance instead of launching RPC server.
No other user changes needed.

```py
from jubakit.classifier import Config, Classifier

cfg = Config()
classifier_service = Classifier.run(cfg, embedded=True)
```

### CODE REVIEW

File: jubakit/base.py [hunk 1]
```diff
@@ -447,23 +448,40 @@ def _client_class(cls):
...
+    #return jubatus.embedded.Classifier
+    raise NotImplementedError()
 
-    # Returns the Service instance.
-    service = cls('127.0.0.1', backend.port)
-    service._backend = backend
+  @classmethod
+  def run(cls, config, port=None, embedded=False):
+    """
+    Runs a new standalone server or embedded instnace and returns the
```
Comment (shiodat) [inline jubakit/base.py]: Typo : `instnance` -> `instance`

File: jubakit/base.py [hunk 2]
```diff
@@ -447,23 +448,40 @@ def _client_class(cls):
...
+    serivce instance.
```
Comment (shiodat) [inline jubakit/base.py]: Typo: `instnace` -> `instance`, `serivce` -> `service`

**Comment by shiodat** [comment]
Great! :+1: 
Could you fix the comment typos?

**Comment by kmaehashi** [comment]
@shiodat Thanks! Fixed typos.

**Comment by shiodat** [comment]
:+1: LGTM

**Pull request merged**